### PR TITLE
Check that wp-config.php exists in order to determine if WP is installed or not

### LINKED ISSUE #80: App shouldn't be considered initialized if wp-config.php missing
### Name and Version

bitnami/wordpress-nginx:5.9.3

### What is the problem this feature will solve?

I am using this image for development, and I would like to mount my plugin development folder to /bitnami/wordpress/wp-content/plugins/my-plugin

When I do so however, the image considers WordPress installed, because the /bitnami/wordpress folder isn't empty:

https://github.com/bitnami/bitnami-docker-wordpress-nginx/blob/master/5/debian-10/prebuildfs/opt/bitnami/scripts/libpersistence.sh#L114

The image tries to restore an existing (nonexistent) app.

### What is the feature you are proposing to solve the problem?

`is_app_initialized` should check whether a wp-config.php file exists in /bitnami/wordpress.

### What alternatives have you considered?

Replacing 5/debian-10/prebuildfs/opt/bitnami/scripts/libpersistence.sh with a custom copy.

### PR DESCRIPTION
**Pull request by ziodave**
**Description of the change**

Extend the check whether WordPress is already installed by looking for a `wp-config.php` file.

**Benefits**

It fixes a problem where the script would incorrectly assume WordPress is installed because the root folder exists. However, the root folder may exist when users mount existing plugins or themes.

**Possible drawbacks**

None.

**Applicable issues**

- Fixes #80

### CODE REVIEW

File: 5/debian-11/prebuildfs/opt/bitnami/scripts/libpersistence.sh [hunk 1]
```diff
@@ -114,7 +114,7 @@ restore_persisted_app() {
 is_app_initialized() {
     local -r app="${1:?missing app}"
     local -r persist_dir="${BITNAMI_VOLUME_DIR}/${app}"
-    if ! is_mounted_dir_empty "$persist_dir"; then
+    if ! is_mounted_dir_empty "$persist_dir" && [[ -f "$persist_dir/wp-config.php" ]]; then
```
Comment (javsalgar) [inline 5/debian-11/prebuildfs/opt/bitnami/scripts/libpersistence.sh:117]: This function is used by all the application in the catalog, could you move the wp-config.php check to libwordpress.sh?

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: 5/debian-11/prebuildfs/opt/bitnami/scripts/libpersistence.sh (hunk 1 above)
Comment (ziodave) [reply 5/debian-11/prebuildfs/opt/bitnami/scripts/libpersistence.sh]: done

**Comment by javsalgar** [comment]
Could you fix the merge issue?

**Comment by ziodave** [comment]
> Could you fix the merge issue?

@javsalgar it's fixed.

**Review by javsalgar — APPROVED** [review]

**Pull request merged**